### ISSUE TITLE
Regularizer/Weight Decay Modification

### ISSUE THREAD

**Issue by quinnZE** [david8862/keras-YOLOv3-model-set#228]
Hello, and thank you for making this repo, it is very helpful. I am trying to make some modification to the weight decay or regularizer on the YOLO v5 models, but I cannot seem to find where the parameters for these are set. Can you point me to the location of these if they are present?

**Comment by david8862** [comment]
@quinnZE  you can check [common/backbones/layers.py](https://github.com/david8862/keras-YOLOv3-model-set/blob/master/common/backbones/layers.py#L10) for regularizer config
